### PR TITLE
restructure quickstart docs 

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
they are now based on a executable script that runs on rhel and fedora without sudo

also the cfme_test.sh script was renamed to cfme_test_via_docker.sh

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit b077f15c964833095829d6332ba212c9748aade9:

`docs/getting_started.rst`:
- :heavy_exclamation_mark: Line 19: **ERROR/3** *No format (Writer name) is associated with this role: "raw".*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 89cfdcd48e054f15ac220c3f157eefccfc584f24:

`docs/getting_started.rst`:
- :warning: Line 50: **WARNING/2** *Explicit markup ends without a blank line; unexpected unindent.*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 122c744583753bcf97e283e95696be9c2112a20f:

`docs/getting_started.rst`:
- :warning: Line 49: **WARNING/2** *Explicit markup ends without a blank line; unexpected unindent.*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5c66757b78ab298ffc19594a7e569c9c40df07ad:

`docs/getting_started.rst`:
- :warning: Line 61: **WARNING/2** *malformed hyperlink target.*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by psav** [comment]
Can we fix this I'd love to get it merged

**Comment by dajoRH** [comment]
Lint report for commit b5e58012a8b1c10c5ab862ae5e3f16306c376a53:

`docs/getting_started.rst`:
- :heavy_exclamation_mark: Line 74: **ERROR/3** *No format (Writer name) is associated with this role: "raw".*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 9c20463da57d5a32f95a5a38f42b99398fdc0e8a:

Everything seems all right :smile: .
*CFME QE Bot*

File: docs/getting_started.rst [hunk 1]
```diff
@@ -12,89 +12,17 @@ setup and execution scripts.
 
 Setup
 -----
-You can use this shortcut to install the system and python dependencies which will leave you only
-with the need to copy the yamls and putting the ``.yaml_key`` in place. Copy this to an executable
-file, place it in the ``cfme_tests`` repository (along ``conftest.py``):
```
Comment (jkrocil) [inline docs/getting_started.rst]: We want to keep the `.yaml_key` information in the guide for sure.

File: docs/getting_started.rst [hunk 2]
```diff
@@ -106,50 +34,51 @@ will not work.
    `cd /path/to/virtualenv'
    source bin/activate or . bin/activate
 
-* Install python dependencies:
-
-  * ``PYCURL_SSL_LIBRARY=nss pip install -Ur /path/to/virtualenv/cfme_tests/requirements.txt``
-  * If you get error from pycurl and you used this command, you might like to remove pycurl and try
```
Comment (jkrocil) [inline docs/getting_started.rst]: The pycurl-related info is valuable as well; we had people run into this issue multiple times before.

**Comment by jkrocil** [comment]
@mfalesni I remember you being concerned about parts of the guide as well. Was it the part about symlinks in `conf`?

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by mfalesni** [comment]
@jkrocil For me it is important that the guide contains all the required informations, that is why I am opposed to deleting of them right now. Like the ENV variables and so. With regard to symlinks, I don't remember any specific thing but if you are encrypting the credentials, in order to preserve the symlinks, the encrypt_conf.py should be used instead of the ipython juggling.

**Comment by RonnyPfannschmidt** [comment]
closing this one in favor of #2956 due to documentation problems

**Pull request closed without merging**